backend/lib/calculate/adequacy.py

```diff
@@ -198,5 +206,7 @@ def calculate_adequacies(
             )
         )
 
+    adequacies_response = _add_closest_provider_id(adequacies_response, location_mapping)
```

Comment:
I object to calling this variable `adequacies_response` until it is in the same format as `api/responses`.